packages/opencensus-instrumentation-mongodb/src/mongodb.ts

```diff
@@ -0,0 +1,162 @@
+/**
+ * Copyright 2018, OpenCensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {types} from '@opencensus/opencensus-core';
+import {classes} from '@opencensus/opencensus-core';
+import {logger} from '@opencensus/opencensus-core';
+import * as mongodb from 'mongodb';
+import * as semver from 'semver';
+import * as shimmer from 'shimmer';
+import {callbackify} from 'util';
```

Comment:
FYI, `util.callbackify` is not available in Node 6.

Reply:
That was an old dependency that I let trought. My mistake.